lib/rubocop/cop/rspec/expect_actual.rb

```diff
@@ -5,6 +5,10 @@ module Cop
     module RSpec
       # Checks for `expect(...)` calls containing literal values.
       #
+      # Autocorrect is supported, but when both expected and actual values
```

Comment:
WDYT of "Autocorrection is performed when the expected is not a literal"?

Reply:
Thank you so much.That looks better. Changed wording.